indico/modules/events/registration/models/forms.py

```diff
@@ -269,6 +269,12 @@ def get_registration(self, user=None, uuid=None, email=None):
         if user:
             return user.registrations.filter_by(registration_form=self, is_cancelled=False).first()
         if uuid:
-            return Registration.query.with_parent(self).filter_by(uuid=uuid, is_cancelled=False).first()
+            return (Registration.query
+                    .with_parent(self)
+                    .filter_by(uuid=uuid, is_cancelled=False, is_deleted=False)
```

Comment:
https://bitbucket.org/zzzeek/sqlalchemy/issue/3308/cannot-use-hybrid_property-in-filter_by